Src/FluentAssertions/TypeEnumerableExtensions.cs

```diff
@@ -77,5 +77,60 @@ public static IEnumerable<Type> ThatImplement<T>(this IEnumerable<Type> types)
         {
             return new TypeSelector(types).ThatImplement<T>();
         }
+
+        /// <summary>
+        /// Filters to only include types that are classes.
+        /// </summary>
+        public static IEnumerable<Type> ThatAreClasses(this IEnumerable<Type> types)
+        {
+            return new TypeSelector(types).ThatAreClasses();
+        }
+
+        /// <summary>
+        /// Filters to only include types that are not classes.
+        /// </summary>
+        public static IEnumerable<Type> ThatAreNotClasses(this IEnumerable<Type> types)
+        {
+            return new TypeSelector(types).ThatAreNotClasses();
+        }
+        /// <summary>
+        /// Filters to only include types that are static classes.
+        /// </summary>
+        public static IEnumerable<Type> ThatAreStatic(this IEnumerable<Type> types)
+        {
+            return new TypeSelector(types).ThatAreStatic();
+        }
+
+        /// <summary>
+        /// Filters to only include types that are not static classes.
+        /// </summary>
+        public static IEnumerable<Type> ThatAreNotStatic(this IEnumerable<Type> types)
+        {
+            return new TypeSelector(types).ThatAreNotStatic();
+        }
+
+        /// <summary>
+        /// Filters to only include types that satisfies the <paramref name="predicate"/> passed.
+        /// </summary>
+        public static IEnumerable<Type> ThatAre(this IEnumerable<Type> types, Func<Type, bool> predicate)
+        {
+            return new TypeSelector(types).ThatAre(predicate);
+        }
+
+        /// <summary>
+        /// Returns T for the types which are Task&lt;T&gt; or ValueTask&lt;T&gt;; the type itself otherwise
```

Comment:
```suggestion
        /// Returns T for the types which are <see cref="Task{T}" /> or <see cref="ValueTask{T}" />; the type itself otherwise
```
The same for the other uses of `&lt;` in this PR.

Reply:
Fixed